checkmate/app.py

```diff
@@ -16,6 +18,11 @@ def load_settings(settings):
     :raise ValueError: If a required parameter is not filled
     :return: A dict of settings
     """
+    for param in OPTIONAL_PARAMS:  # pragma: no cover
+        value = settings.get(param, os.environ.get(param.upper()))
+        if value:
+            settings[param] = value
```

Comment:
Because we didn't have a DB in production, so this couldn't be deployed. This probably can be done now. This has been a sequencing nightmare.

Reply:
Quite a long answer, but basically:

 * At the time of writing we didn't have a DB
 * Even if we do have a DB, we don't have a way of updating it yet as the script will be run on multiple machines (we need celery stuff)
 * Therefore in order to not do, _everything_ in one massive commit a split needs to be made
 * This can (and must) be deployed out without the DB

We can of course just not deploy any of it until it's all done, but this is going to either force us to either:

* Not merge anything until it's all done
* Block other potential bugs and issues behind it
* Maintain a separate branch or something icky